### ISSUE TITLE
Can't get past "Listen for Event" for Unity source setup

### ISSUE THREAD

**Issue by kyleqian** [amplitude/unity-plugin#91]
I'm trying to do the initial setup steps for the Unity SDK to connect it with my account, but I can't get past the part of the setup where I log my first event.

## Expected Behavior
Amplitude setup page receives the log event from my editor, updates, and lets me continue setting up my project.

## Current Behavior
Nothing happens.

## Possible Solution
n/a

## Steps to Reproduce
1. Log in Monobehaviour, in editor:
```
    private async void Awake() {
        var amplitude_ = Amplitude.getInstance();
        amplitude_.setServerUrl("https://api2.amplitude.com");
        amplitude_.logging = true;
        amplitude_.trackSessionEvents(true);
        amplitude_.init("5b80fd3d07b69c5c2ef65d0af190793f");
        amplitude_.logEvent("EVENT_NAME_HERE"); // Confirmed this logs "C# sendEvent EVENT_NAME_HERE" into console
   |
```
2.  See `C# sendEvent EVENT_NAME_HERE` and `C# init 5b80fd3d07b69c5c2ef65d0af190793f` in the Unity console.
3. Nothing else happens.

## Environment
- Unity Plugin Version: 2.2.2 and 2.2.0
- Device: Macbook Pro
- Device OS and Version: Big Sur 11.4 / set to iOS deployment target

**Comment by dantetam** [comment]
Hello @kyleqian ,

1) Confirm that you are using the correct API key. 
2) Sometimes the events are slow to update. On the left sidebar of the Amplitude website, click on `User & Account Look-Up`
![Screen Shot 2021-09-22 at 8 38 14 AM](https://user-images.githubusercontent.com/4561104/134375717-4eeaadb4-6e0e-47d6-804e-c2653c34d94e.png)

**Comment by kyleqian** [comment]
I wasn't able to see any of the dashboard until I set up a working data source, so I just did it with the HTTP api via curl. That worked, which confirms that the key is correct and that my computer is capable of sending log events (so probably the issue is local to my Unity setup?).

On the Unity setup page I still see this, without any change:
![Screen Shot 2021-09-22 at 11 25 57](https://user-images.githubusercontent.com/6733773/134401409-bd80ac4f-d784-453b-8501-eea21226d179.png)

I see this on my dashboard (the one event there is the one that went through testing the HTTP api):
![Screen Shot 2021-09-22 at 11 26 16](https://user-images.githubusercontent.com/6733773/134401525-9a69122f-36b4-469c-9f84-ce96f9c5199b.png)

This is my Unity console:
<img width="711" alt="Screen Shot 2021-09-22 at 11 33 30" src="https://user-images.githubusercontent.com/6733773/134401754-30f3b3bd-4ad3-466f-8d5f-0addc455fc4f.png">

**Comment by dantetam** [comment]
Hello @kyleqian ,

You should configure the app to build on Android or iOS. Our Unity SDK does not work for pure desktop yet. See `Build Settings` and then `Switch Platform`. You see should output in logcat or xcode for the respective mobile platform

**Comment by kyleqian** [comment]
The project is already set for iOS - but you're saying it isn't possible to test in the editor, and that it only works once I build to a device? Is there no way to make it work in the editor besides, say, using the HTTP api then?

**Comment by dantetam** [comment]
Yes, do not test in the Unity editor. On say a MacBook with an iOS Unity app, you may emulate a device to test on with xcode. Otherwise, I assume you are some other platform, and need to test some other way.

**Comment by kyleqian** [comment]
I see, I think that needs to be clearer in the setup, that you can't do this from the editor:
![Screen Shot 2021-09-22 at 12 45 55](https://user-images.githubusercontent.com/6733773/134411509-af201d7d-ca67-4f07-9eb3-a1bbd29b1c72.png)

**Comment by dantetam** [comment]
I agree with you. I'll update this source as well as the Developer Docs for Unity. Will close this issue when I'm done

**Comment by dantetam** [comment]
It's been fixed some time ago. Thank you for the callout @kyleqian

**Comment by SirPax** [comment]
I'm having the same problem but with Android.
**AND** when running from Android device (not in editor).

console says "event sent" but the setup page keeps at "Listening...".

Any help would be greatly appreciated.
